x-pack/plugins/security_solution/common/endpoint/generate_data.ts

```diff
@@ -1579,45 +1579,7 @@ export class EndpointDocGenerator extends BaseDataGenerator {
    * Generates a Fleet `package policy` that includes the Endpoint Policy data
    */
   public generatePolicyPackagePolicy(): PolicyData {
-    const created = new Date(Date.now() - 8.64e7).toISOString(); // 24h ago
-    // FIXME: remove and use new FleetPackagePolicyGenerator (#2262)
-    return {
-      id: this.seededUUIDv4(),
-      name: 'Endpoint Policy',
-      description: 'Policy to protect the worlds data',
-      created_at: created,
-      created_by: 'elastic',
-      updated_at: new Date().toISOString(),
-      updated_by: 'elastic',
-      policy_id: this.seededUUIDv4(),
-      enabled: true,
-      inputs: [
-        {
-          type: 'endpoint',
-          enabled: true,
-          streams: [],
-          config: {
-            artifact_manifest: {
-              value: {
-                manifest_version: '1.0.0',
-                schema_version: 'v1',
-                artifacts: {},
-              },
-            },
-            policy: {
-              value: policyFactory(),
-            },
-          },
-        },
-      ],
-      namespace: 'default',
-      package: {
-        name: 'endpoint',
-        title: 'Elastic Endpoint',
-        version: '1.0.0',
-      },
-      revision: 1,
-    };
+    return new FleetPackagePolicyGenerator('seed').generateEndpointPackagePolicy();
```

Comment:
this test fails with the `generateEndpointPackagePolicy`, I don't see what's wrong: https://buildkiteartifacts.com/e0f3970e-3a75-4621-919f-e6c773e2bb12/0fda5127-f57f-42fb-8e5a-146b3d535916/01900103-f516-4ed4-87c4-9b7d61018d23/0190011d-efc2-4ae5-b798-183cd412c094/target/test_failures/0190011d-efc2-4ae5-b798-183cd412c094_5229d8d2e85e30fb77d09b2cb7ac453b.html?response-content-type=text%2Fhtml&X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=ASIAQPCP3C7L5GFNCXL7%2F20240610%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20240610T092750Z&X-Amz-Expires=600&X-Amz-Security-Token=IQoJb3JpZ2luX2VjEKf%2F%2F%2F%2F%2F%2F%2F%2F%2F%2FwEaCXVzLWVhc3QtMSJHMEUCIQCUHrtisFsQkvOlwftKOykqTIAZWlhP1WiGCZrmpGuoSAIgZma%2B4nKHNqr%2Bw1z3z3LR4IvveMr24JTO1ElisyIXtaQq8QMIQBAAGgwwMzIzNzk3MDUzMDMiDGQO5yUCNsTkMMF4JCrOA8REXR%2BMIJ377ev3qmApy5nSRsY8YIqUHbunbRUMpBRVaeB4VICaU5CkVuH2F1gzmLF7eeb4S0XRkInqT3XADvG4W4iqj7YNSjV0r%2Bgcn0gLqASsVex4hz6bLYxBmglYKxX54WEY5yr4ZBZc9zPZNoUv9g%2Bv8LTqORmULH%2Fssgz2%2FPH6fh7RZEoG%2FOXIMhxiSsbIYzeDUsxwBtLvr5MHiTyROH%2BOGyoeA80sdWluBleFa%2F2v327NaUEB8bRD1w5AzYbwYYm%2FLrKJ4x4Oy7rmC9tE3bJtSVjmW0IxzabGd6JkTXtBDUvrx2moCw0uJo7p8EGrNHgF2%2FbNGZFYIlIOzyEh3TR3GOJlibXPN%2B7I%2BzNziAwZy3U%2BoicrB%2BlBjMES4SQdx%2B0pGf%2FoUGzwDPrQ9eXc8pqLdlfQLcmc0SN%2B4Dh9h9%2F6eLoj5nXvR3%2BejHReJnKUVNWuA5W9r49lX%2Fo8btL69ZIz3AX60lKEcMs9Zz5pPT7qhJLklHBSyatrT%2BGzjh9rk3i0XRZi1vz8kCAONa81YOASnsrq3OVR%2FEATm7tJ5Y6ZdBRqR%2FDd55rkOt4X1J1RqLVoBWk%2BI4uVF4zwpaL6Q8OccIwjbofvIrIU3TDRwZqzBjqlAVASWNwKfemspPUGVlOc02JckXyMCDuevh7xQgIUC5t7%2FvTQ3jNS4zu4Mprv%2FfUk8xMhbBM3IpHWrbmnl5ByAxnWaAMBm%2FO695%2FyYIzCIPxATzQr%2BTxvM6mvjtiD2XaNYzZyUcVrteQH6sB23H96lww71niN3ZyHHAQGNzOIkZeIqc1BB87DbzWRXzkI%2B7hG%2BlE4A9smMg9%2FcUXQ8fQ7erYk2GYcSA%3D%3D&X-Amz-SignedHeaders=host&X-Amz-Signature=d6b3f69cf031b90b322f01643d8f4fb3fc5ddd440c1943dae6c6e4f1f810029b